puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/updateJobs.groovy

```diff
@@ -0,0 +1,20 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Update Jenkins Jobs') {
+            steps {
+                deleteDir()
+                git url: 'https://github.com/theforeman/foreman-infra.git'
```

Comment:
A few folks have indicated that the strategy to follow is more efficient than a deleteDir and re-clone:

```
                checkout([
                    $class : 'GitSCM',
                    branches : [[name: '*/tfm-ror52']],
                    extensions: [[$class: 'CleanCheckout']],
                    userRemoteConfigs: [
                        [url: 'https://github.com/theforeman/rails-packaging']
                    ]
                ])
```